### DISCUSSION TITLE
Android client - distribution and updates

### DISCUSSION THREAD

**Discussion by arnowelzel** [rustdesk/rustdesk#8620] [General]
I installed the Android client using F-Droid from here: https://f-droid.org/de/packages/com.carriez.flutter_hbb/

However that version is 1.2.3 from 2023-11-22 and outdated and my client alway tells me to download the new version.

The latest release is 1.2.6. I don't know why there is no newer version available on F-Droid since the new release is already 2 weeks old and there is a tag for it at https://github.com/rustdesk/rustdesk/releases/tag/1.2.6.

Since F-Droid is not updated yet, I tried to use the package from https://github.com/rustdesk/rustdesk/releases/tag/1.2.6 - but neither AArch64 (ARM64) nor ARMv7 (32-bit) work on my Google Pixel 6a, it alway says "invalid package".

I understand, that in the past you had issues with criminals abusing RustDesk and ask their victims to install it from Google Play - but at least app updates work way better using this way. Maybe you consider Google Play again with a clear warning in the app description, that you should not install the app if asked by people you don't know as this is may be a security risk.

In addition you may also implement some kind of monitoring on your public relay server. Under normal circumstance no user would establish 100 or more of client sessions within a day from the same IP address and without a paid license, so this could be used to block potentially malicious users - since they don't buy a license but still try to convince their victims to let them see their screen using RustDesk.

**Comment by arnowelzel** [comment]
I just learned, that you can't update the client using the release package from GitHub when you originally installed it using F-Droid since the code signing keys of the packages don't match. After removing the client I was able to install the latest version from GitHub. But this is a really cumbersome way.

**Reply by basilgello** [reply]
https://github.com/rustdesk/rustdesk/discussions/8544
